### PR TITLE
Meenu/fix: added bi to career page

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/data/_career-data.ts [hunk 1]
```diff
@@ -730,6 +730,69 @@ const career_data = {
...
+                {
+                    title: 'Senior Data Analyst',
+                    location: 'Limassol, Cyprus',
+                    id: '47807000001891704',
+                },
+                {
+                    title: 'Senior Data Engineer',
+                    location: 'Limassol, Cyprus',
+                    id: '47807000001891732',
+                },
```
Comment (habib-deriv) [inline src/pages/careers/data/_career-data.ts:771]: Let's add 4 items. Rest of them can be visible from the view zoho page

File: src/pages/careers/_join_your_team.tsx [hunk 2]
```diff
@@ -195,6 +195,14 @@ const JoinYourTeam = () => {
             tagline:
                 'We take care of collaboration details to ensure that projects are completed in a timely, efficient manner.',
         },
+        {
+            img_data: data.projectManagement,
```
Comment (habib-deriv) [inline src/pages/careers/_join_your_team.tsx:199]: Let's have a separate image for BI team, if not available in design need to get from UX team

Comment (meenakshi-deriv) [reply src/pages/careers/_join_your_team.tsx:199]: sure @habib-deriv i will check with them.

File: src/pages/careers/data/_career-data.ts (hunk 1 above)
Comment (meenakshi-deriv) [reply src/pages/careers/data/_career-data.ts:771]: Will make the changes @habib-deriv . Thanks

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by NikitK-deriv — APPROVED** [review]

**Pull request merged**